include/flyft/mesh.h

```diff
@@ -72,12 +71,19 @@ class Mesh
         
         bool operator==(const Mesh& other) const;
         bool operator!=(const Mesh& other) const;
+        
+        BoundaryType getlower_boundary_type() const;
+        BoundaryType getupper_boundary_type() const; 
```

Comment:
These should be added after the other getters, after line 51. For consistency with other methods in this class, they should be named like:
```suggestion
        BoundaryType lower_boundary_condition() const;
        BoundaryType upper_boundary_condition() const; 
```
We can revisit in a later PR whether we want to unify the naming convention for methods across classes to use more conventional get/set.